libs/beacon/src/beacon_service.cpp

```diff
@@ -159,11 +239,29 @@ BeaconService::State BeaconService::OnWaitForSetupCompletionState()
   return State::WAIT_FOR_SETUP_COMPLETION;
 }
 
+// Attempt to determine whether the node is attempting to generate entropy that is far in the past
+// (possible on start up when syncing)
+bool BeaconService::OutOfSync()
```

Comment:
Now that we might push an old aeon into the beacon service, it needs a way to realise it has fallen behind and leave the state machine